EasyVCR.Tests/ClientTest.cs

```diff
@@ -127,6 +127,39 @@ public async Task TestCensors()
             Assert.AreEqual(censoredHeader, censorString);
         }
 
+        [TestMethod]
+        public async Task TestRegexCensors()
+        {
+            var cassette = TestUtils.GetCassette("test_regex_censors");
+            cassette.Erase(); // Erase cassette before recording
+
+            // set up regex pattern
+            var url = new Uri(FakeDataService.GetIPAddressDataUrl());
+            var domain = url.Host;
+            var regexPattern = domain;
+
+            // set up advanced settings
+            const string censorString = "censored-by-test";
+            var advancedSettings = new AdvancedSettings
+            {
+                Censors = new Censors(censorString).CensorPathElementsByPatterns(new List<string> { regexPattern })
+            };
+
+            // record cassette with advanced settings
+            var client = HttpClients.NewHttpClient(cassette, Mode.Record, advancedSettings);
+            var fakeDataService = new FakeJsonDataService(client);
+            var _ = await fakeDataService.GetIPAddressDataRawResponse();
+
+            // manually check that the cassette contains the censored path element
+
+            // verify that censoring does not interfere with replay
```

Comment:
These comments should be connected to something? They're floating alone so it's hard to know if they are accidentally still leftover from something or should be associated to a block/line of code?

Reply:
They are separate, the top one does not accompany any code (as it describes)

Reply:
Hmm, then we should move the bottom comment to be attached to the associated code in question and the top comment should probably instead be a docblock or something for this test describing to the user (not clarifying code inline) that this test requires some manual verification